test/gtest/uct/test_peer_failure.cc

```diff
@@ -568,6 +573,20 @@ class test_uct_peer_failure_keepalive : public test_uct_peer_failure
 
         test_uct_peer_failure::kill_receiver();
     }
+
+#ifdef HAVE_IB
```

Comment:
maybe add to test_rc instead?

Reply:
there are 2 options:
1. ifdef HAVE_IB for IB
2. copy most of peer-failure stub into test_rc

option 1 is just simpler

Reply:
IMO move to test_rc (maybe inherit from common class)

Reply:
okay... PR became 3x longer :)